src/adventures-photography/_posts/2017-08-10-eurovelo-montenegro.md

```diff
@@ -0,0 +1,71 @@
+
+We spent much of the day riding above small towns and villages on a busy road which was clearly built to link all of the towns. We thought about leaving it for the smaller roads, but quickly agreed the undulations would be an absolute killer, so opted to stay on the main road as the drivers were — generally — being respectful. 
+
+{% figure %}
+  {% img src: 2016-06-web-DSCF7496, alt: Statue in Macedonia. %}
+
+  <div class="row pair">
+    {% img src: 2016-06-web-DSCF7499, alt: Skopje Bizzare. %}
+    {% img src: 2016-06-web-DSCF7500, alt: Flags, bridges and statues in Macedonia. %}
+  </div>
+
+  {% img src: 2016-06-web-DSCF7505, alt: Statue in Macedonia. %}
+
+  {% figcaption %}**Top:** A small village on the Montenegro coast of unknown name; **Middle, left:** tunnel into Budva; **Middle, right:** Sveti Nikola Island, Budva; **Bottom:** On the ferry across the Bay of Kotor{% endfigcaption %}
+{% endfigure %}
+
+As we approached Budva we diverted onto the old road which had been closed for cars and descended gently to a tunnel through a headland before popping out on the waterfront in town. We followed this until we located some lunch, and then located the road the the Bay of Kotor ferry crossing. Once the initial climb was out of the way this proved to make for easy riding which quickly levelled out. 
+
+Once we’d crossed the by a on the ferry we managed to avoid any significant climbing until we hit the border crossing. Here we passed through the first checkpoint, before proceeding to climb for over a kilometre before we arrived at the Croatian side of the border checkpoint. Here we joined the queue for EU citizens and were waved through after nothing more than a glance at the cover of our passports. 
+
+Locating somewhere to stop for a few minutes we had a look at the map and found a campsite not far off route right on the seafront. We pulled off the main road and started following the smaller lane towards the village of Molunat. 
+
+Our first night in Croatia turned out to be an entertaining one. On arrival we went through the usual routine to getting our gear sorted before Max went off to swim in the sea (which after a minute he decided was “too salty”) while I went to shoot the bay. When we got back we found another person around our own age setting up a tent next to ours who was walking the Croatia coast on his own. 
+
+We spent a few hours chatting about our own adventures before it started getting dark and decided it was time to go and get food, as we stood to leave another cyclist turned up who we started talking with. She was on a solo cycling expedition around Europe and travelling under the name of _The Daring Dutchie_. She was clearly tired and hungry so we helped her set up her dinner (while Jamie went off to find some footie) and then went off in search of dinner and a group. Three main meals later it was time for bed. 
```

Comment:
Are there places you could link to here?

Also: this is the first time you mention "Jamie".

Reply:
Sadly not, I was hopeful that I would be able to 😞

Reply:
Ah, dammit.

I'd introduce the first chap as "Jamie", but otherwise that'll be it.

Reply:
Cool, thanks Nick. I'll get this squared after work.